packages/core/components/Alert/components/Alert.tsx

```diff
@@ -0,0 +1,36 @@
+import { IoMdCheckmarkCircleOutline, IoIosCloseCircle, IoIosInformationCircle } from 'react-icons/io'
```

Comment:
why aren't we reusing our Icon component?

Reply:
Do you see an issue with using react-icons? They provide an easy way to add icons and offer icons that COVE doesn't and it looks like the library is already installed.

Reply:
It's another thing we have to maintain... <Icon /> is much more widely used. Does react-icon do anything other than having different icons? We can extend <Icon /> for all of those use cases.

Reply:
I think using a dedicated icon library might be the better choice but I can update this.

Reply:
I would be fine with using this library if we deprecate the Icon component and add, at a minimum, a comment to it saying to not use it. You'll also need to update the storybook guides that describe how to use Icon component.

Reply:
I think the Icons we have in COVE are supplemental. For example, chart icons that we want to provide that aren't standard in Icon libraries. Either way, I've gone ahead and updated this component to use the COVE `<Icon />` component now.

Reply:
I think the references aren't being used and can be taken out now right?

Reply:
Removed the references now.